etc/conf/ccTelemetry.yaml

```diff
@@ -709,3 +709,15 @@ playback_keywords:
  - HIERARCH SIMULATED MJD-OBS
  - HIERARCH SIMULATED DATE-OBS
  - HIERARCH SIMULATED DATE-END
+
+# Keywords not to ignore from telemetry in playback mode
+playback_keywords_keep:
+  - EMUIMAGE
+  - 'DATE-OBS'
```

Comment:
Are the single quotes here for a special reason? The `HIERARCH` lines above don't have quotes and they contain words with dashes.

Reply:
I was worried that the `-` was going to mess up the string. But now I know that it wont't.